csm_web/scheduler/admin.py

```diff
@@ -88,7 +88,8 @@ class StudentAdmin(CoordAdmin):
     list_display = ("name", "get_email", "get_course", "get_mentor", "section")
     search_fields = ("user__email", "user__first_name", "user__last_name")
     actions = ("drop_students", "undrop_students")
-    autocomplete_fields = ("section", "user")
+    #autocomplete_fields = ("section", "user") TODO: restore this
```

Comment:
we can probably more or less do away with this altogether since we now have the coord interface

Reply:
I'll leave it commented out just in case for now, but I agree with you.